README.md

```diff
@@ -22,24 +22,21 @@ Start the app by executing the following.
 $ npm start
 ```
 
-If you want the UI to use a local version of the backend, you need to set the following environment variables:
+The frontend will use your localhost backend as default. 
+If you want the UI to use another version of the backend, you need to set the following environment variables:
 
 ```bash   
-$ export LAUNCHER_BACKEND_URL=http://localhost:8080/api/
```

Comment:
I think it's fine to leave `localhost' here, since it's the most common used copy-pasted snippet